composer.json

```diff
@@ -36,8 +36,8 @@
         "silex/silex": "~2.0",
         "symfony/process": "~2.1|~3.0",
         "guzzlehttp/psr7": "^1.3",
-        "php-http/curl-client": "^1.5",
-        "php-http/mock-client": "^1.0"
+        "php-http/mock-client": "^1.0",        
+        "php-http/guzzle6-adapter": "^1.1.1"
```

Comment:
I have replaced php-http/curl-client with php-http/guzzle6-adapter because PSR-7 aware client is required.